demo/os/armv8a/bin/embed

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Same comment -- these all look like the x86_64 versions.  Suggest to combine into a common "grub-arch".